framework/src/postprocessors/InterfaceDiffusiveFluxIntegral.C

```diff
@@ -72,10 +66,9 @@ InterfaceDiffusiveFluxIntegralTempl<is_ad>::InterfaceDiffusiveFluxIntegralTempl(
                  "neighbor_variable should be of a similar variable type.");
 
   // Warn that we are not using the same gradient to compute the diffusive flux here
-  if (_fv && (_fv_variable == _neighbor_fv_variable))
+  if (_fv && !isParamValid("neighbor_variable"))
     mooseWarning("Only one finite volume variable was specified. InterfaceDiffusiveFluxIntegral is "
-                 "only accurate at a "
-                 "FVDiffusionInterface, not with a regular diffusion kernel.");
+                 "only accurate at a FVDiffusionInterface, not with a regular diffusion kernel.");
```

Comment:
I tend to think we should straight-up error instead of potentially producing inaccurate results